Mathlib/Topology/Algebra/ConstMulAction.lean

```diff
@@ -240,6 +240,19 @@ theorem Dense.smul (c : G) {s : Set α} (hs : Dense s) : Dense (c • s) := by
 theorem interior_smul (c : G) (s : Set α) : interior (c • s) = c • interior s :=
   ((Homeomorph.smul c).image_interior s).symm
 
+@[to_additive]
+theorem IsOpen.smul_left {s : Set G} {t : Set α} (ht : IsOpen t) : IsOpen (s • t) := by
+  rw [← iUnion_smul_set]
+  exact isOpen_biUnion fun a _ => ht.smul _
+
+@[to_additive]
+theorem subset_interior_smul_right {s : Set G} {t : Set α} : s • interior t ⊆ interior (s • t) :=
+  interior_maximal (Set.smul_subset_smul_left interior_subset) isOpen_interior.smul_left
+
+@[to_additive (attr := simp)]
+theorem smul_mem_nhds {t : Set α} (g : G) {a : α} : g • t ∈ 𝓝 (g • a) ↔ t ∈ 𝓝 a :=
```

Comment:
Should this be `set_smul_mem_nhds_smul_iff`? The one below could gain a subscript zero suffix.

Reply:
We have both `set_smul` and `smul_set`. `Data/Set/Pointwise/SMul` uses both. Should we normalize?

Reply:
Ah sorry, I meant `smul_mem_nhds_smul_iff`. I think we can put off normalizing the `set` part

Reply:
What about the new names?